src/main/java/com/ge/predix/uaa/token/lib/AbstractZoneAwareTokenService.java

```diff
@@ -66,9 +67,13 @@
     public OAuth2Authentication loadAuthentication(final String accessToken)
             throws AuthenticationException, InvalidTokenException {
 
+        if (this.getServiceBaseDomain().isEmpty() && this.getServiceZoneHeadersList().isEmpty()) {
+            throw new InvalidRequestException(
+                    "Service configuration error: Service not providing base domain or header parameter");
```

Comment:
"ZacTokenService requires atleast one of the following properties to be configured: serviceBaseDomain or serviceZoneHeaders ."